src/index.html

```diff
@@ -3,13 +3,17 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>HTML-CSS-Practice-two</title>
+    <title>HTML-CSS-Practice-1</title>
```

Comment:
This HTML-CSS-Practice-two is correct. No need to fix it. Please revert it to previous